### PR TITLE
Section7

### PR DESCRIPTION
**Pull request by PNQT**

### CODE REVIEW

File: src/layout/main.html [hunk 1]
```diff
@@ -286,6 +289,64 @@ <h6>Get discounted coupons</h6>
         </div>
         <img src="assets/images/desktop/background.png" alt="">
     </section>
+
+    <section class="testimonials">
+         <figure class="testimonials_layer">
+            <img src="assets/icons/layer_1.svg" alt="">
```
Comment (PhamNhuQuocBao) [inline src/layout/main.html:295]: do not leave the alt value empty. let fix all.

File: src/layout/main.html [hunk 2]
```diff
@@ -286,6 +289,64 @@ <h6>Get discounted coupons</h6>
...
+                    <h5 class="testimonials_main_content_header_title">
+                        Testimonials 
+                    </h5>
+                    <h3 class="testimonials_main_content_header_name">
+                        Trust our clients
+                    </h3>
+                </div> 
+     
+                <figure class="testimonials_main_content_image">
+                    <img src="assets/icons/ellipse 22.svg" alt="">
```
Comment (PhamNhuQuocBao) [inline src/layout/main.html:312]: avoid naming files with spaces, I have already mentioned this issue.

File: src/stylesheet/pages/home/_testimonials.scss
Comment (PhamNhuQuocBao) [inline src/stylesheet/pages/home/_testimonials.scss:1]: format code.

**Review by PhamNhuQuocBao — CHANGES_REQUESTED** [review]

**Comment by datmaiz** [comment]
resolve conflicts, please

**Pull request merged**